### ISSUE TITLE
Missing form label on searchbar

### ISSUE THREAD

**Issue by blue-belle** [FlorOps/hdg-wiki-two#6]
WAVE accessibility tool says there's a missing form label on the search bar. I assume an aria-label that's missing?

**Comment by blue-belle** [comment]
<img width="1133" alt="Image" src="https://github.com/user-attachments/assets/04248365-37ed-4879-af96-b4b58b8486f1" />

relevant html
```html
<input placeholder="Search..." class="hdg-search search-input hx-block hx-w-full hx-appearance-none hx-rounded-lg hx-px-3 hx-py-2 hx-transition-colors hx-text-base hx-leading-tight md:hx-text-sm hx-bg-black/[.05] dark:hx-bg-gray-50/10 focus:hx-bg-white dark:focus:hx-bg-dark placeholder:hx-text-gray-500 dark:placeholder:hx-text-gray-400 contrast-more:hx-border contrast-more:hx-border-current" type="search" spellcheck="false">
```
